scripts/dbm.py

```diff
@@ -29,7 +30,12 @@
     '/data/system/gesture.key',
     '/data/system/cm_gesture.key',
     '/data/system/locksettings.db',
-    '/data/system/password.key'
+    '/data/system/password.key',
+    '/data/data/com.android.contacts/databases/contacts.db',
+    '/data/data/com.samsung.advp.imssettings/databases/ims.db',

```

Comment:
Is this specific to Samsung only?
use generalised location for this which can run on every android device

Reply:
Yes, I was testing these yesterday.
Now I committed and removed those dbs locations that were specific for Samsung.